src/platform/macos.rs

```diff
@@ -515,7 +515,7 @@ impl SystemTrayExtMacOS for SystemTray {
     self.0.icon_is_template = is_template
   }
 
-  fn set_icon_as_template(&mut self, enable: bool) {
+  fn set_menu_on_left_click(&mut self, enable: bool) {
```

Comment:
It's good to me but please default with left click. Official tray icons are default with both clicks or left click.
None of them is right click only.

Reply:
yeah that's the default, there was a typo, fixed in ed34d29